.github/ISSUE_TEMPLATE/b-bug-report.yml

```
@@ -0,0 +1,87 @@
+name: Report a bug
+description: You encountered a bug in the geolocator plugin that you would like to report.
+title: "Bug: "
```

nit: common convention seems to be to add square brackets around the `Bug` keyword. Not sure if this has additional benefits though.

```suggestion
title: "[Bug]: "
```